File: app/src/androidTest/java/com/battlelancer/seriesguide/modules/TestTmdbModule.java
package com.battlelancer.seriesguide.modules;

import com.uwetrottmann.tmdb2.Tmdb;
import com.uwetrottmann.tmdb2.services.FindService;
import com.uwetrottmann.tmdb2.services.PeopleService;
import com.uwetrottmann.tmdb2.services.SearchService;
import com.uwetrottmann.tmdb2.services.TvService;

public class TestTmdbModule extends TmdbModule {

    @Override
    FindService provideFindService(Tmdb tmdb) {
        return null;
    }

    @Override
    PeopleService providePeopleService(Tmdb tmdb) {
        return null;
    }

    @Override
    SearchService provideSearchService(Tmdb tmdb) {
        return null;
    }

    @Override
    TvService provideTvService(Tmdb tmdb) {
        return null;
    }
}

File: app/src/androidTest/java/com/battlelancer/seriesguide/modules/TestTmdbModule.kt
package com.battlelancer.seriesguide.modules

import com.uwetrottmann.tmdb2.Tmdb
import com.uwetrottmann.tmdb2.services.PeopleService
import com.uwetrottmann.tmdb2.services.SearchService
import com.uwetrottmann.tmdb2.services.TvService

class TestTmdbModule : TmdbModule() {
    override fun providePeopleService(tmdb: Tmdb): PeopleService? {
        return null
    }

    override fun provideSearchService(tmdb: Tmdb): SearchService? {
        return null
    }

    override fun provideTvService(tmdb: Tmdb): TvService? {
        return null
    }
}